dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -34,6 +34,10 @@
             required="false" type="Boolean" default="true"/>
          <AD id="customClaims" name="%customClaims" description="%customClaims.desc"
             required="false" type="String" cardinality="2147483647"/>            
+         <AD id="thirdPartyIDTokenClaims" name="thirdPartyIDTokenClaims" description="thirdPartyIDTokenClaims.desc"
+            required="false" type="String" cardinality="2147483647"/>
+         <AD id="thirdPartyAccessTokenClaims" name="thirdPartyAccessTokenClaims" description="thirdPartyAccessTokenClaims.desc"
+            required="false" type="String" cardinality="2147483647"/>
```

Comment:
Both of these need to be marked `ibm:beta="true"` until we have testing in place for them.